client/src/pages/About.tsx

```diff
@@ -39,28 +39,50 @@ export const About = () => {
         <p><strong>Damien:</strong> Rock chameleon, from garage to hard rock, he digests it all and spits out good music.</p>
       </section>
 
-      <section className={styles.section}>
+      <section className={styles.githubContainer}>
         <p>You can also see our Github links !</p>
+        <div className={styles.github}>
           <a
           href="https://github.com/Salvak613"
           target="_blank"
           rel="noopener noreferrer"
-          ><strong>Jules</strong></a>
+          >
+          <img 
+          src="https://avatars.githubusercontent.com/u/201595920?v=4"
+          alt="Avatar of a cat">
```

Comment:
rather put the name of the github user rather than the description of the actual pp